composites/basic/Tabs.js

```diff
@@ -0,0 +1,92 @@
+import React from "react";
+import styled from "styled-components";
+import PropTypes from "prop-types";
+
+const Tab = styled.section`
+	padding: 18px 20px;
+	display: none;
+	
+	&.tab-active {
+		display: block;
+	}
+`;
+
+/**
+ * Creates a series of tabs.
+ */
+class Tabs extends React.Component {
+	/**
+	 * Constructs the class and sets its initial state.
+	 *
+	 * @param {Object} props The props to use within the component.
+	 *
+	 * @constructor
+	 */
+	constructor( props ) {
+		super( props );
+
+		this.state = { activeTab: this.props.activeTab };
```

Comment:
Why set the state to a new value each time you get a new prop? Why not use the prop itself?